ChangeLog.md

```diff
@@ -1,5 +1,17 @@
 # ChangeLog hie-bios
 
+## 2020-08-26 - 0.7.0
+
+### New Features
+
+* Allow specifying a stack.yaml for stack configurations [#230](https://github.com/mpickering/hie-bios/pull/230)
+  * Changes type of
```

Comment:
Is it complete?

Reply:
should be removed, I added that to the section "API Changes"